apps/crunchy-postgres/workload/run_litmus_test.yml

```diff
@@ -0,0 +1,63 @@
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cruncy-postgres-loadgen 
+  namespace: litmus
+spec:
+  template:
+    metadata:
+      name: crunchy_postgres-loadgen
+      namespace: litmus
+      labels:
+        app: crunchy-postgres-loadgen
+    spec:
+      serviceAccountName: litmus
+      restartPolicy: Never
+      containers:
+      - name: loadgen
+        image: openebs/ansible-runner:ci
+        imagePullPolicy: Always
+        env: 
+          
+          - name: ANSIBLE_STDOUT_CALLBACK
+            #value: log_plays
+            value: default
+
+          - name: LOADGEN_LABEL
+            value: loadgen=crunchy-loadgen
+             
+            #namespace in which loadgen will run 
+          - name: LOADGEN_NAMESPACE
+            value: litmus
+    
+            # Service name of postgres application 
+          - name: SERVICE_NAME
+            value: pgset
+    
+            # Database Name 
+          - name: DATABASE_NAME
+            value: postgres
+    
+           # Password to access Database
+          - name: PASSWORD
+            value: password
+    
+           # Database user
+          - name: DATABASE_USER
+            value: testuser
+          
+            #Port on which crunchy databse is listening
+          - name: PORT
+            value: "5432"
+            
+            #Number of parallel transactions to perform
+          - name: PARALLEL_TRANSACTION
+            value: "5"
+            
+            #Number of transaction to perform
+          - name: TRANSACTIONS
+            value: "200"  
+
+        command: ["/bin/bash"]
+        args: ["-c", "ansible-playbook ./crunchy-postgres/workload/test.yml -i /etc/ansible/hosts -v; exit 0"]
```

Comment:
@shashank855 , can we get application label as variable to derive service name/IP address rather than obtaining it as input from user?

Reply:
@gprasath this change is not required. what happen if the user doesn't have a label for service.